extra/man/alacritty.5.scd

```diff
@@ -534,6 +534,20 @@ This section documents the *[cursor]* table of the configuration file.
 
 	Default: _0.15_
 
+# Terminal
+
+This section documents the *[terminal]* table of the configuration file.
+
+*osc52* "Disabled" | "OnlyCopy" | "OnlyPaste" | "CopyPaste"
+
+	Controls the ability to write to the system clipboard with the _OSC 52_
+	escape sequence. While this escape sequence is useful to copy contents
+	from the remote server, the ability to paste is not that necessary
+	given that normal `Paste` action could be used instead and such
+	sequence could be abused.
```

Comment:
```suggestion
	Controls the ability to write to the system clipboard with the _OSC 52_
	escape sequence. While this escape sequence is useful to copy contents
	from the remote server, allowing any application to read from the clipboard can be easily abused while not providing significant benefits over explicitly pasting text.
```

My main gripe with this comment was the "not that necessary" part sounding kinda imprecise. I think the new suggestion should be a little more direct.